lib/active_merchant/billing/gateways/quickbooks.rb

```diff
@@ -78,7 +78,7 @@ def authorize(money, payment, options = {})
         post[:capture] = 'false'
 
         response = commit(ENDPOINT, post)
-        check_token_response(response, ENDPOINT, post)
+        check_token_response(response, ENDPOINT, options, post)
```

Comment:
Minor quibble but the usual argument pattern has `options`  last.

Reply:
good call, updated! 👍